internal/routes.go

```diff
@@ -249,7 +249,7 @@ func fetchPerson(id string, personCh chan Person) {
 	person := Person{}
 	errDecode := json.NewDecoder(res.Body).Decode(&person)
 	if errDecode != nil {
-		fmt.Println("Failed to decode person: ", errDecode, " with ID: ", id)
+		log.Println("Failed to decode person: ", errDecode, " with ID: ", id)
```

Comment:
I think we should add how we react to the error as well, as in the weird `Id : -1` skip hack. I think I wrote ` - Will skip`

Reply:
Done